pandas/tests/io/test_sql.py

```diff
@@ -938,6 +938,12 @@ def test_database_uri_string(self):
 
         # using driver that will not be installed on Travis to trigger error
         # in sqlalchemy.create_engine -> test passing of this error to user
```

Comment:
@neirbowj ahh I see what you posted on the issue. ok then your fix is fine (just need the noqa), and put an explanation of what is going on here.